storage/innobase/dict/dict0load.cc

```diff
@@ -1897,6 +1897,7 @@ dict_load_index_low(
 	(*index)->page = mach_read_from_4(field);
 	ut_ad((*index)->page);
 	(*index)->merge_threshold = merge_threshold & ((1U << 6) - 1);
+	(*index)->trx_scn = 0;
```

Comment:
`dict_mem_index_create()` will invoke `mem_heap_zalloc()` to create a zero-initialized object. I do not think that the caller of `dict_process_sys_indexes_rec()` would refer to the field `trx_scn`. At least I do not see any change to `i_s_sys_indexes_fill_table()` in this pull request.